dpytools/email/ses/client.py

```diff
@@ -0,0 +1,86 @@
+import re
+import boto3
+from botocore.exceptions import BotoCoreError, ClientError
+
+class SesClient:
+    """
+    A client for sending emails using AWS SES and the boto3 client.
+
+    Attributes:
+        client: The boto3 SES client.
+        sender: The sender's email address.
+    """
+
+    EMAIL_REGEX = r"(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$)"
+
+    def __init__(self, sender: str, aws_region: str):
+        """
+        Initialise the SesClient with the sender's email and AWS region.
+
+        Args:
+            sender (str): The sender's email address.
+            aws_region (str): The AWS region.
+
+        Raises:
+            ValueError: If the sender's email or AWS region is invalid.
+        """
+        # check sender is actually a valid email (as in the format)
+        if not re.match(self.EMAIL_REGEX, sender):
+            raise ValueError("Invalid sender email format")
```

Comment:
show the thing that isn't valid please. Have a hard look at any error message your write without any kind of variable in it, it's very, very rare that you'll want a generic error message.